README.md

```diff
@@ -387,6 +389,25 @@ defradb start --allowed-origins=http://localhost:3000
 
 The catch-all `*` is also a valid origin. 
 
+## Backing up and restoring
```

Comment:
thought: I like this backing up and restoring analogy better :), but wondering if should be consistent as currently we have files with the following names:
- `backup_export.go`
- `backup_import.go`

Maybe instead can be:
- `backup.go`
- `restore.go`

Reply:
The way I went about it is: it's the backup feature and it can export and import data. I wanted a category of actions.

Reply:
Yep I get that, I feel like backup is being used as an umbrella term. IMO commands will be simpler too if we can pick up one concise word for each action. As pointed out below